packages/esm-tb-app/src/views/program-management/tabs/mdr-tb.component.tsx

```diff
@@ -14,13 +14,13 @@ const MdrTbList: React.FC<PatientChartProps> = ({ patientUuid }) => {
   const { t } = useTranslation();
   const config = useConfig();
   const headerTitle = t('MdrTbEnrolment', 'TB/MDR TB Enrolment');
-  const [isEmptyOutcome, setIsEmptyOutcome] = useState(true);
+  const [isEmptyOutcome, setIsEmptyOutcome] = useState(false);
 
   useEffect(() => {
     fetchPatientLastEncounter(patientUuid, config.encounterTypes.tbProgramEnrollment).then((encounter) => {
       const result = encounter?.obs?.filter((ob) => ob?.concept?.uuid === config.obsConcepts.outcome);
-      if (result?.length) {
-        setIsEmptyOutcome(false);
+      if (result?.length === 0) {
+        setIsEmptyOutcome(true);
```

Comment:
Unless am misunderstanding this but this translates to hide the form launcher if there no records. So won't this stop the user from entering new records?

Reply:
@pirupius No actually. That was the behaviour before this change.